app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CartsController < ApplicationController
```

Comment:
Nice that you created a separate controller for the Cart functionality. Good job!